# Data Management and Shared Resources {#datamanagement}
This section will describe how the Lippmann Lab stores and manages data. 

## Storage and Documentation 
### Lab Shared Network Drive
Our lab has a shared network drive folder where members of the lab can store raw data. You should have requested access to this folder in the [Accessing Lab Accounts and Spaces]{#access} section of this document. The address of this folder is smb://vu1file.it.vanderbilt.edu/lippmannlab/. In order to log in, you will need to type "VANDERBILT/Your NetID" for the username and your normal Vanderbilt password for the first password. If it is your first time logging in, you should set up a folder with your name. This is a communal resource, so everyone has access to your files once you've uploaded them to the drive. Make sure you are saving your files to this location from the Leica microscope or you are using your own external drive as storage space on the Leica computer is limited. Some members of the lab also use this as backup for important documents such as qualifying exams, fellowship applications, etc. 

#### Other Shared Access Drives
Many [core facilities](#corefacilities) also have their own folders for storing and managing data. You can check each core facility's website to find the address for its specific folder. Additionally, the Bellan lab has it's own folder for saving confocal data; you will need to ask a Bellan lab member for access to this remote folder. 

#### Remote Access Using Pulse Virtual Private Network (VPN)
If you want to access data saved in these folders off-campus, you will need to set up [Pulse VPN](https://it.vanderbilt.edu/services/catalog/end-point_computing/network_access/remote-access/index.php). Note that you will need [Duo Multi-Factor Authentification](#duo) set up to use the Pulse VPN service. If you are on campus and connected to the Vanderbilt WiFi, you should be able to open these folders without connecting to the VPN. 

### Google Drive {#googledrive}
The [lab Google Drive folder](https://drive.google.com/drive/folders/0Bwvn9S-4oeMmeHhzUS1nM0FNak0?usp=sharing) is another storage site commonly used to share information with the entire lab. This information includes locations of reagents, protocols, and example fellowship/qualifying exam documents. Lab members generally do not use this space to store data. You should be updating these files if you move or receive reagents or if you develop any protocols for the lab. 

### Vanderbilt Box {#vanderbiltbox}
[Vanderbilt Box](https://vanderbilt.account.box.com/login) is where most data and document sharing with Ethan occurs. You should use your normal VU NetID and password to log into this account. If this is your fist time logging in, you should make a folder entitled "[Your Name] Research" and share it with Ethan. This is where research updates, subgroup meeting presentations, raw paper data, etc. is stored. ***Once you have submitted a paper, make sure that Ethan has all the raw data in a Box folder in order to comply with RCR standards!*** [Subgroup meeting presentations](#subgroup) and lab meeting presentations should also be stored in your box folder for future reference. 

### GitHub {#github}
The [Lippmann Lab GitHub](https://github.com/lippmannlab) is where code sharing occurs. There are different repositories for different programs that lab members use to quantify their data or control their bioreactors. Generally, the repositories that involve quantifying data stores a README file that explains how to install and use the code, all the code needed to run the program, and an example file to run the code with. If you edit any part of these repositories, make sure your commit messages are detailed enough that others can understand what changes you have made. If you are creating a new repository, follow the "Neuron Image Processor" as a template for what should be included. Public repositories such as the Spinfinity code does not require a GitHub account to access, but any private repositories will require that you create an account and that a lab member adds you as a collaborator to the project. 

### Computer Back Ups
You should be backing up your computer relatively frequently. This can take many forms but can be extremely useful if your computer crashes or you accidentally delete data. 

## Reference Management {#zotero}
Most members of the lab use [Zotero](https://www.zotero.org/) as their reference manager. You should [download](https://www.zotero.org/download/) the program to your computer at the beginning of your graduate training. Additionally, it is helpful to install the Chrome connector, which allows you to easily save papers that you are reading to Zotero. Furthermore, Zotero has plugins for both [Word](https://www.zotero.org/support/word_processor_plugin_usage) and [Google Docs](https://www.zotero.org/support/google_docs) that will allow you to insert both citations and bibliographies effortlessly into papers. Zotero allows you to format your citations and bibliographies in a variety of different citation styles, and you can easily convert your settings in the same document. 

Another free reference manager that some lab members use is [Mendeley](https://www.mendeley.com/download-desktop-new/), although it is much easier to have everyone using the same program when writing papers in groups. 

Furthermore, this [Excel Journal Database](https://www.youtube.com/watch?v=Y40Wy1guAiQ) may also be a good tool for keeping up with what papers you have read during your training, although it should not be used as your citation manager. 

## Communication 
This section will contain tips on how to present scientific work. An entire chapter is dedicated to writing manuscripts. 

### Creating an Oral Presentation {#oralpres}
All graduate students and postdocs will present a research update once a semester at the lab group meeting. Additionally, lab members may have the opportunity to speak at department retreats, seminars, and conferences. 

- **Make sure you don't put too much content on each slide.** The goal is for your audience to listen to you as you speak and not spend the entire time trying to read your slide. Figures are preferable over text. Large or complicated figures should be broken down. You can use basic animations to make the information contained in the slides, but don't get too cute with your use of animations. You can also split complicated topics into a couple slides. Animations or pointers can also help your audience know what you are discussing on each slide. 

- **Make sure your slide titles are descriptive and contain the main takeway.** Don't use slide titles that don't add any information or is too technical. 

- **Citations should be enough information for you audience to find the articles/websites.** This is usually the fist author, journal title, and year at the minimum for articles. You should cite all figures that are not originally made by you. 

- **Make sure your text is large enough to be seen from far way.**

- **Figues should be fully explained.** You should describe what the data points are, what the axes mean, etc. in addition to explaining the main takeway from the figure. 

- **Remember that anything you put on your slide is fair game.** The audience has the right to ask you questions on anything that you put on your slide. This is especially true for qualifying exams and dissertation defenses. Be careful putting in information or figures that you are not prepared to explain in detail. 

- **It's usually a good idea to pause briefly before moving on to the next slide in order to see if anyone has any questions.** You can also incorporate planned pauses into your presentations in order to make room for audience questions. 

- **Be aware of who your audience is when you are creating your presentations.** The way that you present in a lab meeting might be different than how you present at a conference. Make sure you know who might be in your audience when you are putting it together so that you don't get too technical and detailed or too broad and superficial. This includes knowing when you should define abbreviations. For example, you might not need to define iPSCs or BBB in lab meetings, but you should define it at a qualifying exam or at a conference. 

- **If you are using a pointer, make sure you practice using it before you present.**

- **[Practice](#practice) your oral presentation with the lab!** If you are presenting at a conference/meeting or for a department milestone, it is highly encouraged that you practice with the lab. Just send out an email or Slack message about availability, and senior students would be happy to watch a practice presentation. 

#### Journal Club Presentations {#journalclub}
The lab rotates who presents an article for our lab's journal club, which is usually emailed out at the beginning of each semester. There are also other opportunities to present at other journal clubs on campus such as at the Alzheimer's Disease Journal Club. In addition to the tips from the above section on oral presentations, here are some additional tips for presenting at journal clubs. 

- **Picking an article to present:** You should choose a paper that has been published relatively recently. Ethan will typically assist younger students before their first presentation. These articles should be sent to lab at least a week prior to presenting. 

- **Know background material about the chosen paper.** You should have knowledge of the research and work that contributed to the paper you chose to present. You should provide enough information in the introduction for lab members who may not be familiar with the topics described in the paper. Simplify complex topics for non-experts to understand.

- **Describe the research question and the importance/context of the research.** This can also help organize your presentation as the authors draw conclusions to answer these research questions.

- **Break down figures and other complex information.** The way data is presented in a paper is not necessarily the way it should be presented in an oral presentation. Break down figures into smaller, understandable chunks so that it's not too overwhelming to your audience. Additionally, make sure you make the figures large enough for your audience to read. Be ready to discuss all the components of the figures. Try not to make your slides too text heavy. 

- **Provide information on uncommon techniques used in the paper.**

- **Be ready to discuss the paper in lab.** Identify pros and cons about the techniques used and/or the conclusions drawn from the paper. What questions did you still have after reading the paper? This helps show an understanding of the paper, and what you learn from this practice can help with peer reviewing papers. Additionally, it can lead to better lab discussions about the paper. 

- **Be ready to discuss why you chose that paper.** You can talk about how it relates to your work or techniques you are interested in using. 

### Creating a Poster Presentation 
Lab members may also present their work in poster at internal and external meetings and conferences. 

- **Know who your audience is when creating your poster.** This will help determine how much background information needs to be provided as well as what jargon you should use. Your poster should not be too technical and detailed if you are presenting to a broad conference, but it should also not be too broad or simplified if you are presenting at a field-specific conference. 
- **Your posters should contain at minimum:** an introduction/background/significance section, at least one results/discussion/methods section, a conclusion/future directions section, an acknowledgments section, and a reference section. 

- **Figures and diagrams are preferable to text, but you should give enough information that your audience can understand your figures and the conclusions that are drawn from your results.** 

- **Use bullet points with short sentences/phrases rather than paragraphs of text for easier reading.** Make sure any text is concise and relevant. Use consistent formatting throughout the poster. Try to avoid redundancy and vague phrases. Make sure your results headings draw attention to the question you are answering and/or conclusions you are drawing from the results.

- **Make sure figures and text are legible from far away.** 

- **Use blank space wisely.** This empty space can help draw attention to important information. Scale and color can also demonstrate relative importance. 

- **Practice how you are going to present your poster to your audience.** 

- **Senior students would be happy to look over posters and provide feedback.** 

- Poster templates can be found [here](https://drive.google.com/drive/folders/1V1ykFkQy5sY_4hgSlZhh6y-axiy3mreH?usp=sharing). You can download these templates and edit it for your own posters. 

- You can refer to the [Better poster movement](https://www.insidehighered.com/news/2019/06/24/theres-movement-better-scientific-posters-are-they-really-better) for additional tips on creating a poster presentation. 

## Computation/Data Analysis
### Accessing Software
We use a variety of software for analyzing data. [R](https://www.r-project.org/) and [RStudio](https://rstudio.com/products/rstudio/download/) are open source and free to download. [Fiji/ImageJ](https://imagej.net/Fiji) is an open source tool for processing images. Other sofware such as Microsoft Office and Matlab can be downloaded for free from [Vanderbilt IT Services](https://it.vanderbilt.edu/software-store/). Other software can also be bought at a discount through Vanderbilt IT. You can discuss any software needs with Ethan in order to see if the lab should buy a license.

### Norms for Analysis
- Our lab uses [this Excel spreadsheet](https://drive.google.com/file/d/1eRxUonvqSrtfm6mePRuh6x_J_fUgkLIo/view?usp=sharing) as a template for analyzing TEER data. 

- Make sure scale bars are sufficiently large enough to be seen if the image is small. You should have a scale bar in at least one image if you have a series of images that are at the same scale. 

### Documentation 
- **Protocol generation and storage occurs in our lab's [Google Drive Folder](#googledrive).**
    - Make sure you cite any papers that your protocols might be based off of. 
    
    - Make sure you note any edits you make to existing protocols. 
    
    - All protocols should contain a list of reagents (with product numbers) and reagent preparation/storage information in addition to the protocol. 
    
    - Include graphics as necessary to understand the timeline of different differentiation protocols. You can also include markers that are used to verify cell identities as applicable. 
    
    - Add a sufficient amount of information in your protocol so that your lab members can follow the protocol with minimal guidance. 

- **Code documentation is stored in our lab's [Github](#github).** 
    - Make sure your commits are informative and explain what you changed and why.
    
    - Make sure you provide an example of how to run your code. 
    
    - Make sure you describe how to install your code and what the inputs/outputs should be. 

### Tools
- **Learning to Code**
    - [CZI](https://chanzuckerberg.com/ndcn/member-login/?redirect_to=https://chanzuckerberg.com/ndcn/) holds weekly [Computational Biology Office Hours](https://docs.google.com/spreadsheets/d/10Euplhsqu_e9fZAnuadgKFWKtDzElxS664gwxAsWvbg/edit#gid=213176177) to address computational needs in the neurodegeneration community. They can also hold Hackathons to specifically address any computational problems that a lab might want to solve. The CZI Neurodegeneration website also has a section where you can access any computational resources created in the community. 
    
    - [Choose youR own Pythway](https://openscapes.github.io/cyop/)
    
    - [R for Excel Users](https://rstudio-conf-2020.github.io/r-for-excel/)
    
    - [R for Data Science](https://r4ds.had.co.nz/)
    
    - [Statistics and R](https://www.edx.org/course/statistics-and-r)
    
    - [Python Data Science Handbook](https://jakevdp.github.io/PythonDataScienceHandbook/)
    
    - [Really Friendly Command Line Intro](https://hellowebbooks.com/learn-command-line/)
    
    - [Intro to ImageJ](https://imagej.net/Introduction)
    
    - [Carpentries Sofware Lessons](https://software-carpentry.org/lessons/) and [Carpentries Data Lessons](https://datacarpentry.org/lessons/)

- **RNAseq**
    - [Analysis of single cell RNA-seq data](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html)
    
    - [Current best-practices in single-cell RNA-seq: a tutorial](https://github.com/theislab/single-cell-tutorial)
    
    - [Getting Started with Cell Ranger (10X Platform)](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/using/tutorial_ov)
